prepare_shop/action.yaml

```diff
@@ -275,73 +229,28 @@ runs:
         echo -e "\033[0;35m###  Cloning ${REF} ###\033[0m"
         git clone --depth 2 "${URL}" --branch "${REF}" --single-branch source
 
-    - name: Clone the shop (PR ${{ inputs.github_event_number }})
-      if: ${{ inputs.compilation_url == '' && (inputs.github_event_name == 'pull_request' && github.repository == 'OXID-eSales/oxideshop_ce') }}
+    - name: 'PR-Clone ${{ inputs.git_repository }} (${{ inputs.git_ref }})'
+      if: ${{ inputs.compilation_url == '' && inputs.github_event_name == 'pull_request' }}
       shell: bash
       run: |
-        # prepare_shop: Clone the shop (PR)
+        # prepare_shop: Clone the shop ${{ inputs.git_repository }} (${{ inputs.git_ref }})
```

Comment:
Same here, the "name" and the comment should match.

Reply:
`'PR-Clone ${{ inputs.git_repository }} (${{ inputs.git_ref }})'`
!=
`Clone the shop ${{ inputs.git_repository }} (${{ inputs.git_ref }})`